src/platforms/node/connect.mdx

```diff
@@ -9,11 +9,12 @@ If you are using `yarn` or `npm`, you can add our package as a dependency:
 
 ```bash
 # Using yarn
-$ yarn add @sentry/node@5.20.1
+$ yarn add @sentry/node@5.21.4
```

Comment:
I don’t think we should specify version numbers to make sure we don’t have to keep updating this

Reply:
Fair. And agreed. It felt like more than I was going to do in the GH UI, LOL... but I suppose there's no reason we couldn’t just take the version off entirely. Will do that.